### PR TITLE
Added differences between SR and JK Flip-flop, D and T Flip-flop

### LINKED ISSUE #86: Difference between SR and JK flip-flop
Feature request:
There no explanation of difference between SR and JK flip-flop
There no explanation of difference between D and T flip-flop

### PR DESCRIPTION
**Pull request by Prerna-0202**
Fixes #86 

#### Changes done:
-  I have added differences between SR and JK Flip-flop, D and T Flip-flop as per issue #86 

#### Screenshots
Screenshot (9)
Screenshot (11)

#### ✅️ By submitting this PR, I have verified the following

- [ ] Checked to see if a similar PR has already been opened 🤔️
- [ ] Reviewed the contributing guidelines 🔍️
- [ ] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [ ] Tried Squashing the commits into one

### CODE REVIEW

**Comment by Prerna-0202** [comment]
@manjotsidhu, Can you please review my pr once and provide me with feedback ...

**Review by manjotsidhu — APPROVED** [review]
Implementation looks good, we need review the quality of content.

**Comment by Prerna-0202** [comment]
Okay @manjotsidhu Sir, Thanks for reviewing my pr...

File: docs/seq-ssi/flip-flops.md [hunk 1]
```diff
@@ -281,6 +281,23 @@ Q(n+1) = Q(n)'J + Q(n)K'
 ````
 <iframe width="100%" height="400px" src="https://circuitverse.org/simulator/embed/47630" id="masterslave_jk_flipflop_01" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
 
+## Differences between SR and JK flip-flop
+
+| SR Flip-flop      |  JK Flip-flop     |
+|:------------|:--------------|
+| When J input is 1 and K input is 0 on the triggering edge of clock pulse,then output is in set state | When S input is 1 and R input is 0,output is in set state |
```
Comment (ayan-biswas0412) [inline docs/seq-ssi/flip-flops.md:288]: I think the contents of SR Flip-flop and JK flip flop got reversed here.
Or you need to swap the heading of the table.

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]

File: docs/seq-ssi/flip-flops.md (hunk 1 above)
Comment (Prerna-0202) [reply docs/seq-ssi/flip-flops.md:288]: @ayan-biswas0412, Yes sir it's by mistake, I have commit the changes. Can you please review it again...

**Review by ayan-biswas0412 — APPROVED** [review]
Looks fine to me. 🚀
Good work @Prerna-0202 🎉

**Comment by Prerna-0202** [comment]
@ayan-biswas0412, Thanks for reviewing !!!

**Comment by tachyons** [comment]
Reverting as it seems that some contents are copied from [here](http://www.cburch.com/logisim/docs/2.1.0/libs/mem/flipflops.html)

**Pull request merged**